src/parsers/directive.parser.ts

```diff
@@ -18,7 +18,10 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		template = this._normalizeTemplateAttributes(template);
 		$(template)
 			.find('[translate]')
-			.each((i: number, element: CheerioElement) => {
+            .contents() // get child nodes
+            .filter(function() { // only keep text nodes
+				return this.nodeType === 3; // node type 3 = text node
```

Comment:
it's important to filter by node type text, otherwise we'll use the full content of the DOM node.
For example: 

```html
<div translate>
  <span class="icon">&times;</span>
  Completed
</div>
```

Without filtering you would extract `&times;\n\r                      Completed`, with filtering you only extract `Completed`. That's how the directive works as well.